From 2297b10f5e003fc34ea8da368e3a1452ed2e46a4 Mon Sep 17 00:00:00 2001
From: Andrei Simescu <andrei.simescu@nxp.com>
Date: Fri, 3 Jul 2026 16:07:31 +0300
Subject: [PATCH] TARS-37: Specify python version to use for venv in install.sh

Previously, the install.sh script used uv venv to create the connector's
runtime venv. However, this will default to the host installed python version,
which might not be usable for the connector.

This commit explicitly specifies a supported version, ensuring compatibility
regardless of host python version.

Upstream-Status: Pending
Signed-off-by: Andrei Simescu <andrei.simescu@nxp.com>
---
 install.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install.sh b/install.sh
index 7e6693e..7602623 100755
--- a/install.sh
+++ b/install.sh
@@ -186,7 +186,7 @@ fi
 section "Setting up eIQ AAF Connector environment"

 info "Creating python venv (/usr/share/eiq/aaf-connector/venv)..."
-uv venv "/usr/share/eiq/aaf-connector/venv"
+uv venv --python 3.13 "/usr/share/eiq/aaf-connector/venv"
 source "/usr/share/eiq/aaf-connector/venv/bin/activate"

 section "Installing dependencies in venv"
--
2.41.0

